modular_skyrat/code/modules/reagents/chemistry/reagents/drink_reagents.dm

```diff
@@ -0,0 +1,152 @@
+//maint energy and subtypes
+/datum/reagent/consumable/maint_energy
+	name = "Maintenance Energy"
+	description = "Assistant's favorite energy drink."
+	nutriment_factor = 0.5 * REAGENTS_METABOLISM //the original flavor, now with 0% less sugar.
+	color = "#00014d60"
+	taste_description = "raw energy"
+	glass_icon_state = null
+	pH = 5 //slightly acidic
+	glass_name = "glass of Maintenance Energy"
+	glass_desc = "Unleash the tide."
+	var/list/healed_roles = list("Assistant")
+	var/heal_amount = 0.25
```

Comment:
Maybe nerf a little.

Reply:
It's half of what doctor's delight is capable of healing, i think it's fine.